go.mod

```diff
@@ -27,6 +27,7 @@ require (
 	github.com/gin-contrib/size v0.0.0-20220707104239-f5a650759656
 	github.com/gin-gonic/contrib v0.0.0-20201101042839-6a891bf89f19
 	github.com/gin-gonic/gin v1.8.1
+	github.com/gobuffalo/packr v1.30.1
```

Comment:
Needs a `make gomodtidy`

Reply:
Doesn't help, pushed manually cleaned up version, for some reason `go.mod` keep changing for me even with `make gomodtidy`